scripts/travis/configure-make.sh

```diff
@@ -0,0 +1,35 @@
+# raw Makefile configuration
+
+LINE () {
+  echo "$@" >> Makefile.config
+}
+
+cp Makefile.config.example Makefile.config
+
+LINE "BLAS := open"
+LINE "WITH_PYTHON_LAYER := 1"
+
+if $WITH_PYTHON3 ; then
+  # XXX doesn't work
```

Comment:
For notes like this we like to use `TODO(handle)` like the `TODO(shelhamer)` you see in the Caffe code. `XXX` isn't currently used in the project for highlighting comments.